publicsuffix-list/PKGBUILD

```diff
@@ -4,6 +4,7 @@ pkgname=publicsuffix-list
 _gitcommit=ffb052a26672745790ec8a3f78c50c03e4dc935d
 pkgver=20170929.553.ffb052a
 pkgrel=1
+epoch=1
```

Comment:
Git for Windows does not modify `publicsuffix-list`. Therefore this change should be unnecessary.